src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -55,16 +57,15 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
 
     async execute(orderRequest: OrderRequestBody, options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
         const { payment, ...order } = orderRequest;
-        let paymentResult: PaymentIntent | StripePaymentMethod;
-        let paymentPayload: any;
+        let formattedPayload: any;
```

Comment:
Any?

Maybe you can at least use something like `let formattedPayload: { }`; But I would rather recommend to create an interface for it.

Reply:
If you don't mind, we'll raise a new ticket to address this because `formattedPayload` has a strange structure and needs further investigation and testing. 😬 @cbandam

Reply:
I think Victor referred in a create an specific type for this scenario, not for all formattedPayloads, maybe something like StripeV3FormattedPayload or some similar of that, Or Do I have a different idea @vmparra  ?